src/frontend_workspaces/frontend/src/carbon-chat/customSendMessage.ts

```
@@ -609,6 +609,7 @@ export async function customSendMessage(
 
           let answerText = accumulatedText || "";
           let answerSources: MessageSource[] = [];
+          let answerMemoryUsage: { count: number; entityIds: string[] } | null = null;
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)